src/store/modules/splitFlow.js

```diff
@@ -104,7 +139,141 @@ const mutations = {
       availableSlot.id = primarySlot.id;
     }
     primarySlot.id = id;
-  }
+  },
+  setSyncMode(state, payload) {
+    if (payload) {
+      //Force the second slot to be the new viewer in payload and change the 
+      //view to the payload's layout
+      let secondSlot = state.slotInfo.find(slot => slot.name === "second");
+      if (payload.flag === true) {
+        let firstSlot = state.slotInfo.find(slot => slot.name === "first");
+        let firstSlotId = firstSlot.id;
+        let originalSlot = state.slotInfo.find(slot => slot.id === 1);
+        secondSlot.id = payload.newId;
+        if (originalSlot.name !== "first")
+          originalSlot.id = firstSlotId;
+        firstSlot.id = 1;
+        state.syncMode = true;
+        state.activeView = payload.layout;
+        state.globalCallback = true;
+      } else {
+        state.activeView = "singlepanel";
+        state.syncMode = false;
+        state.globalCallback = false;
+      }
+    }
+  },
+  closeSlot(state, payload) {
```

Comment:
Is this for closing a pane of the split pane or for removing one of the dynamic components?

Reply:
It is for closing a pane for the split pane. This function reassign the entries to different slots if they are affected by the closing slot. There is another function for removing an entry thus removing an instance of the content vuer.